flocker/control/_protocol.py

```diff
@@ -109,6 +109,11 @@ class NodeStateCommand(Command):
     Used by a convergence agent to update the control service about the
     status of a particular node.
     """
+    # FLOC-1513
+    #
+    # Add another argument, an optional `ListOf(Unicode)` representing
```

Comment:
You can't not special-case them, though.  We have to implement a wire protocol.  We can't (and shouldn't) serialize arbitrary objects.